lib/ccf/archive.py

```diff
@@ -297,6 +297,24 @@ def available_diffusion_unproc_names(self, subject_info):
         name_list = self._get_scan_names_from_full_paths(dir_list)
         return name_list
 
+		
+    def running_status_dir_full_path(self, subject_info):
+        """
+        Full path to the unprocessed diffusion data resource directory
+        """
+        path = self.subject_resources_dir_full_path(subject_info)
+        path += os.sep + 'RunningStatus'
+        return path		
+ 
+    # def available_running_status_dir_full_paths(self, subject_info):
```

Comment:
Just for consistency, let's go ahead and have an `available_running_status_dir_full_paths` method. Notice that for all the resource directories we have an `available_..._dir_full_paths` method. So even though it is probably not currently used. Let's just be consistent about always having such a method.

Reply:
changed